src/EventCell.js

```diff
@@ -106,22 +106,22 @@ class EventCell extends React.Component {
       ...props
     } = this.props;
 
-    let title = get(event, titleAccessor),
-      end = get(event, endAccessor),
-      start = get(event, startAccessor),
-      isAllDay = get(event, props.allDayAccessor),
+    let title = get(event.data, titleAccessor),
```

Comment:
I think this should be changed in the titleAccessor, endAccessor, startAccessor, allDayAccessor, respectively. So instead of `event.data`, create a function that returns `event.data` and pass it into the accessors above. This ensures our calendar does not have to know the shape of the data, only how to get it.

Reply:
the accessors should remain as keys for whatever is in data obj.